solidity/contracts/token/TokenBridgeCctp.sol

```diff
@@ -134,23 +126,164 @@ abstract contract TokenBridgeCctp is
         );
 
         // Receive only if the nonce hasn't been used before
-        if (!_isMessageReceived(cctpMessage)) {
+        if (messageTransmitter.usedNonces(_messageNonce(cctpMessage)) == 0) {
```

Comment:
It did seem nicer to have a named function for this check?

Reply:
this matches the comment better